Source/VMware.vSphereDSC/DSCResources/VDPortGroup.ps1

```diff
@@ -251,16 +310,18 @@ class VDPortGroup : BaseDSC {
         $distributedPortGroupParams.VDSwitch = $distributedSwitch
 
         <#
-        ReferencePortGroup is parameter only for the New-VDPortgroup cmdlet
-        and is not used for the Set-VDPortgroup cmdlet.
+        ReferencePortGroup and VLanId are parameters only for the New-VDPortgroup cmdlet
+        and are not used for the Set-VDPortgroup cmdlet.
         #>
         if (![string]::IsNullOrEmpty($this.ReferenceVDPortGroupName)) { $distributedPortGroupParams.ReferencePortgroup = $this.ReferenceVDPortGroupName }
+        if ($null -ne $this.VLanId) { $distributedPortGroupParams.VlanId = $this.VLanId }
 
         try {
+            Write-VerboseLog -Message $this.CreateVDPortGroupMessage -Arguments @($this.Name, $distributedSwitch.Name)
```

Comment:
Here as well **Write-VerboseLog** will need to be replaced by **WriteLogUtil** after the other PR gets approved.